### PR TITLE
[IMP] pos: add port num for windows os, rem useful tips and check indent

### PR DESCRIPTION
**Pull request by LoredanaLrpz**
Task ID: 3193053

### CODE REVIEW

**Comment by Ysoroko** [comment]
Code changes look good to me

File: content/applications/sales/point_of_sale/payment/worldline.rst [hunk 1]
```diff
@@ -15,8 +15,8 @@ Connect an IoT Box
 ------------------
 
 Connecting a Worldline Payment Terminal to Odoo is a feature that requires an IoT Box. For more
-information on how to connect one to your database, please refer to the
-:doc:`IoT documentation </applications/productivity/iot/config/connect>`.
+information on how to connect one to your database, please refer to the :doc:`IoT documentation
+</applications/productivity/iot/config/connect>`.
```
Comment (jcs-odoo) [inline content/applications/sales/point_of_sale/payment/worldline.rst:19]: This is not necessary.

**Review by jcs-odoo — APPROVED** [review]
@robodoo r+

File: content/applications/sales/point_of_sale/payment/worldline.rst [hunk 2]
```diff
@@ -58,8 +55,9 @@ Hostname
 Port number
 ~~~~~~~~~~~
 
-On the *Port number* screen, enter **"9001"** and click on :menuselection:`OK` (*ECR protocol SSL
-no*) :menuselection:`--> OK`. Click on **Stop** three times; the terminal automatically restarts.
+On the *Port number* screen, enter **9001** (or **9050** for Windows) and click on
```
Comment (lse-odoo) [inline content/applications/sales/point_of_sale/payment/worldline.rst:58]: @LoredanaLrpz 
Hey! We were a bit confused by this part of the documentation, what do you mean by (or ... on Windows). You mean, on a virtual IoT box ? Some clients were confused as they thought it was if their computer is on Windows

**Pull request closed without merging**